humanSubject03/humanSubject03_48dof.urdf

```diff
@@ -1153,159 +1153,159 @@
         <axis xyz="0 1 0" />
     </joint>
 
-	<!-- Sensor  1-->
```

Comment:
I notice that in some of the files the sensor elements are changed too. May be related to the text editor.
To keep it clean, please try to modify just the lower and upper limits without effecting other parts of the file

Reply:
Thanks, I didn't notice that. Yes, it should be something related to text editor, I'll try to clean it.

Reply:
In general I would separate the merge for the fix from the merge for text removing.

Reply:
it seems committing those changes are unavoidable due to the presence of mixed line endings probably due to the `.urdf` generation process. Even if I try to commit the changes directly from the online github editor, I got the following warning and all those lines are commited
<img width="1166" alt="Screenshot 2020-07-07 at 10 50 52" src="https://user-images.githubusercontent.com/35487806/86754529-3d2d2f00-c041-11ea-81a4-47b473038ae8.png">

I would proceed merging the PR as it is, this would fix the line endings for future commits.

Reply:
@claudia-lat @Yeshasvitvs since it seems not trivial to clean the commit further can we procede merging as it is?

Reply:
@lrapetti Go ahead. Sorry for the delay, missed the last comment.

Reply:
great, no worries